_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -146,13 +146,13 @@ This [horizontally scrollable][scrollable] `section` element is not included in
 </style>
 <h1>Our sponsors:</h1>
 <section>
-	<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
-	<img src="/test-assets/shared/eu-logo.svg" alt="EU" />
-	<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
-	<img src="/test-assets/shared/eu-logo.svg" alt="EU" />
-	<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
-	<img src="/test-assets/shared/eu-logo.svg" alt="EU" />
-	<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+	<img src="/test-assets/shared/eu-logo.svg" alt="EU logo" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+	<img src="/test-assets/shared/eu-logo.svg" alt="EU logo" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+	<img src="/test-assets/shared/eu-logo.svg" alt="EU logo" />
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
 </section>
 ```
```

Comment:
From my experience these kind of components are visually managed including logos for a stronger visual impact, but their real purpose is to just list a list of brands.

I don't think we should change these examples including the "logo" word into the image alternative, that is not adding relevant information for the user.

Reply:
Ah, yes, especially in a "our sponsors" list 😅